instrumentation/faraday/README.md

```diff
@@ -19,6 +19,7 @@ To install the instrumentation, call `use` with the name of the instrumentation.
 ```ruby
 OpenTelemetry::SDK.configure do |c|
   c.use 'OpenTelemetry::Instrumentation::Faraday'
+  suppress_internal_instrumentation: true
```

Comment:
I see a related PR from @jterapin renaming the option as `enable_internal_instrumentation`, which is language I think I prefer when naming the options.  

Ideally these options would have symmetric naming and functionality.

https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/1533

Would you be amenable to changing the name?

Reply:
Updated.